recipes/_helper/geo.py

```diff
@@ -133,110 +166,68 @@ def air_geocode(inputs):
     geo (dict): Contains inputs as well as geosupport fields
         See geo_parser() for full list of fields
     """
-    hnum, sname, borough, street_name_1, street_name_2 = (
-        str("" if inputs[k] is None else inputs[k])
-        for k in ("hnum", "sname", "borough", "streetname_1", "streetname_2")
+    hnum, sname, borough, street_name_1, street_name_2, street_name_3 = (
```

Comment:
Where does `street_name_1` and the like get set?  I'm following `street_1`but got lost at `street_name_1`.